prebid-mobile/modules/rendering/android-sdk-integration-pb.md

```diff
@@ -205,29 +249,45 @@ rewardedAdUnit?.show()
 ```
 
 {% capture warning_note %}
-Pay attention that the `loadAd()` should be called on the main thread.
+The `loadAd()` method should be called on the main thread.
 {% endcapture %}
 {% include /alerts/alert_warning.html content=warning_note %}
 
 #### Step 1: Create a Rewarded Ad Unit
 {:.no_toc}
 
-Create the `RewardedAdUnit` object with parameters:
+Create the `RewardedAdUnit` object with the parameters:
 
-* `adUnitId` - an ID of Stored Impression on the Prebid server.
+* `adUnitId` - this is the value of the adunit name in the ad server. e.g. for GAM, it's the adunit name. Other ad servers may call this something different, but the idea is that the buyers want to know which slot in the app is up for auction for targeting and reporting. An example would be "/home/upper-mobile-banner".
```

Comment:
legacy: there is no `AdUnitID` - it should be `configId` as for any ad unit / ad view on this page

Reply:
@YuriyVelichkoPI - how can the adunit be registered if not this way? I assume it's needed for the ad server interface and something ought to be able to set imp.ext.gpid

Reply:
Hi @bretg sorry for the late response, I missed your comments somehow. 

Please review my above [comment](https://github.com/prebid/prebid.github.io/pull/4999/files#r1472630457) today - it describes how to set the gpid